app/assets/javascripts/application.js

```diff
@@ -11,6 +11,7 @@
 // about supported directives.
 //
 //= require jquery
+//= require bootstrap-sprockets
```

Comment:
Nice, you remembered to do this. I had a mentee session last night where he showed me some weird, weird errors he got in heroku when he forgot to do this.